src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BranchesView.cs

```diff
@@ -498,7 +502,21 @@ private void OnCreateGUI()
                     if (createBranch)
                     {
                         GitClient.CreateBranch(newBranchName, selectedNode.Name)
-                            .FinallyInUI((success, e) => { if (success) Refresh(); })
+                            .FinallyInUI((success, e) => {
+                                     if (success)
+                                     {
+                                         Refresh();
+                                     }
+                                     else
+                                     {
+                                         var errorHeader = "fatal: ";
+                                         var errorMessage = e.Message.StartsWith(errorHeader) ? e.Message.Remove(0, errorHeader.Length) : e.Message;
+
+                                         EditorUtility.DisplayDialog(CreateBranchTitle,
+                                             errorMessage,
+                                             Localization.Cancel);
```

Comment:
Alert dialogs that are purely informational should provide an OK button for acceptance. Cancel can be confusing/ambiguous.

https://developer.apple.com/macos/human-interface-guidelines/windows-and-views/alerts/